examples/run_java_springboot/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: run
+
+run:
+	docker run --rm \
+	  -v `pwd`/src:/app/src \
+	  -v `pwd`/pom.xml:/app/pom.xml \
+	  -v maven_cache:/root/.m2/repository \
```

Comment:
Just to be sure "maven_cache" is the folder created at the root level, which contains the Maven Cache ?

Reply:
maven_cache is the name for a docker volume.

Reply:
Could you add a small line which explains that thing in the README? -> examples/run_java_springboot/README.md ?

Reply:
@tsunammis doc added in README, shoud be good !